UnitTests/CommonTestUtils/ConfigureJoinableTaskFactoryAttribute.cs

```diff
@@ -54,7 +54,7 @@ public void AfterTest(ITest test)
         {
             try
             {
-                ThreadHelper.JoinableTaskContext?.Factory.Run(() => ThreadHelper.JoinPendingOperationsAsync());
+                ThreadHelper.JoinableTaskContext.Factory.Run(() => ThreadHelper.JoinPendingOperationsAsync());
```

Comment:
❗️ This null check also cannot change

Reply:
refer to https://github.com/gitextensions/gitextensions/pull/7394#discussion_r347552171

Reply:
😕 This _is_ the `AfterTest` method.

Reply:
https://github.com/gitextensions/gitextensions/pull/7394#discussion_r347552171 second half:

> Regarding exceptions in `AfterTest`.
Maybe yes. Which exception takes precedence? The one thrown in `try { }` or the one thrown in `finally { }`?
So let's be explicit and add:
```
catch (Exception) when (_threadException != null)
{
    // ignore the follow-up exception
}
```
i.e.
```
        public void AfterTest(ITest test)
        {
            try
            {
                try
                {
                    ThreadHelper.JoinableTaskContext.Factory.Run(() => ThreadHelper.JoinPendingOperationsAsync());
                }
                finally
                {
                    ThreadHelper.JoinableTaskContext = null;
                    if (_denyExecutionSynchronizationContext != null)
                    {
                        SynchronizationContext.SetSynchronizationContext(_denyExecutionSynchronizationContext.UnderlyingContext);
                        _denyExecutionSynchronizationContext.ThrowIfSwitchOccurred();
                    }
                }
            }
            catch (Exception) when (_threadException != null)
            {
                // ignore the follow-up exception
            }
            finally
            {
                Application.ThreadException -= HandleApplicationThreadException;
                _threadException?.Throw();
            }
        }
```

Reply:
throws `InvalidOperationException` now if `null`